src/orders/components/OrderShippingMethodEditDialog/OrderShippingMethodEditDialog.tsx

```diff
@@ -68,73 +76,163 @@ const OrderShippingMethodEditDialog = (props: OrderShippingMethodEditDialogProps
         }))
         .sort((x, y) => (x.disabled === y.disabled ? 0 : x.disabled ? 1 : -1))
     : [];
+
+  const currentMethodInChoices = availableChoices.some(c => c.value === shippingMethod);
+
+  const unavailableMethodOption =
+    shippingMethod && !currentMethodInChoices
+      ? [
```

Comment:
can you simplify it?

Reply:
I don't know, maybe, but to me it looks quite clear - it's covering for the situation when the method selected for the order is no longer available.

Reply:
its about multiple conditions and nesting and adding ternary operations with markup, which in general hides the conditions, but neverming, non blocking